src/index.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './containers/App/App';
 import * as serviceWorker from './serviceWorker';
+// import { Provider } from 'react-redux';
+// import { createStore } from 'redux';
+// import { composeWithDevTools } from 'redux-devtools-extension'
+// import rootReducer from './reducers'
+import { BrowserRouter as Router } from 'react-router-dom'
```

Comment:
This importing looks very similar to Python's importing. Do you have to do this in every file? Or just in this index file?

Reply:
This is a heavy import file as it's where the redux "store" is created and the React components are rendered to the DOM.  To answer your question, yes, we need to import certain libraries and files in other components as well.  Sounds like python is similar?